### DISCUSSION TITLE
Move a word without dragging selection

### DISCUSSION THREAD

**Discussion by wwkeyboard** [helix-editor/helix#6976] [Q&A]
How do I move the cursor forward or backward a word without dragging a selection behind? `move_next_word_start` and `move_next_long_word_start` both select the area they jump over, meaning I have to hit escape before doing anything with the moved cursor.

**Comment by Abdillah** [comment]
I think the hackish solution is to define a keymap that contain the move and "collapse" cursor command. Using TOML array.
